apps/ui/src/components/IndicatorVotingPower.vue

```diff
@@ -62,8 +68,10 @@ function handleModalOpen() {
         :network-id="networkId"
         :voting-power-symbol="votingPowerSymbol"
         :voting-powers="props.votingPowers"
+        :voting-power-status="status"
         :final-decimals="decimals"
         @close="modalOpen = false"
+        @get-voting-power="$emit('getVotingPower')"
```

Comment:
```suggestion
        @get-voting-power="emit('getVotingPower')"
```

Reply:
Yes, it does not work without the `$`

Reply:
We'd have to do `const emit = defineEmits()`. Actually not sure what should we prefer, I'm not huge fan of `$emit` myself, but we can leave it like this for now.